phys2bids/phys2bids.py

```diff
@@ -219,11 +219,11 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     # Read file!
     LGR.info(f'Reading the file {infile}')
     if ftype == 'acq':
-        from phys2bids.io import load_acq_ext
-        phys_in = load_acq_ext(infile, chtrig)
+        from phys2bids.io import load_acq
+        phys_in = load_acq(infile, chtrig)
```

Comment:
See the comment on the import of `biored`. If you move the import internally to this function, then this is a good solution. Otherwise, you can move this import to the beginning of the file (but I suggest not to).